src/util/pagination.js

```diff
@@ -0,0 +1,19 @@
+import { range } from 'lodash';
+
+export const getPaginationPages = (currentPage, totalPages) => {
+    const pages = []
+    if (totalPages < 3) {
+        for (let i in range(1, totalPages + 1)) {
+          pages.push(i);
+        }
```

Comment:
ovdje si mogao odmah u if-u kada zavrsisi sa for-om samo da stavis `return pages` i onda nemas potrebu za else-om

Reply:
Vazi, odradicu tako